Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableContentViewController.m

```diff
@@ -260,5 +279,18 @@ - (void)executeStatementAndShowResult:(NSString *)statement completion:(void (^_
     } showFrom:self];
 }
 
+- (void)reloadTableDataFromDB {
+    if (self.databaseManager == nil) {
+        return;
+    }
+    NSArray *rows = [self.databaseManager queryAllDataInTable:self.tableName];
+    NSArray *rowIDs = nil;
```

Comment:
Please add type generics to the NSArray while applicable
e.g. `NSArray<NSString *> *`